### DISCUSSION TITLE
Last online displayed on characters page

### DISCUSSION THREAD

**Discussion by MarkimusDev** [gameserverapp/Platform#425] [Suggestions]
In the characters page, please add a "last online" column, and allow us to sort the results by this metric.

**Comment by mvaessen** [comment]
Hi,

When you click the `status` column on that page it will sort the results based on the last activity. This does mean that someone that recently logged out will show up too.

**Reply by GetDerpy** [reply]
> Hi,
> 
> When you click the `status` column on that page it will sort the results based on the last activity. This does mean that someone that recently logged out will show up too.

This does not function as described. When I click the "status" column it sorts them in a random order. When you hover over the "offline" status it displays how long they were offline, but the column is not sorted by this metric. It only seems to sort by the online/offline status.

(Sorry just realised I replied from the wrong account. I am MarkimusDev)

**Reply by mvaessen** [reply]
Hi, i think this is now fixed. Let me know if this solved the suggestion so we can close it

**Reply by MarkimusDev** [reply]
> Hi, i think this is now fixed. Let me know if this solved the suggestion so we can close it

Confirmed, working as intended. Thanks!
